js/views/modals/Settings/Advanced.js

```diff
@@ -48,6 +49,29 @@ export default class extends baseVw {
     return super.getFormData(subset);
   }
 
+  clickResync() {
+    this.resynchronize();
+  }
+
+  resynchronize() {
+    this.getCachedEl('.js-resync').addClass('processing');
+    this.getCachedEl('.js-resyncComplete').addClass('hide');
+
+    this.resync = $.post(app.getServerUrl('wallet/resyncblockchain'))
+      .always(() => {
+        this.getCachedEl('.js-resync').removeClass('processing');
+      })
+      .fail((xhr) => {
+        const failReason = xhr.responseJSON && xhr.responseJSON.reason || '';
```

Comment:
If you're aborting this guy in remove, then make sure to return out of this fail handler if its aborted, otherwise you'll have the dialog come up.